### ISSUE TITLE
[ISSUE] `databricks_grants` on UC external location empty list error

### ISSUE THREAD

**Issue by npfroiland** [databrickslabs/terraform-provider-databricks#1202]
### Configuration
```hcl
resource "databricks_grants" "poc_external_location_grants" {
  external_location = databricks_external_location.poc_external_location.id
  grant {
    principal  = "admins"
    privileges = ["CREATE TABLE", "READ FILES", "WRITE FILES"]
  }
}
```

### Expected Behavior
1. UC external location grants created.

Note: this to applies `databricks_grants` for both `external_location = ` and `storage_credential = `.

### Actual Behavior
1. `│ Error: cannot read grants: got empty permissions list`

### Steps to Reproduce
1. `terraform apply`

### Terraform and provider versions

```
Terraform v1.1.7
+ provider registry.terraform.io/databrickslabs/databricks v0.5.4
```

**Comment by nkvuong** [comment]
This is due to privilege name mismatch between UC API & provider configuration
`READ FILES` vs. `READ_FILES`

**Comment by nfx** [comment]
@nkvuong can you try it out and make a PR? there's a small mapping that needs changing

**Comment by nfx** [comment]
Fixed in #1240
